tests/integration.rs

```diff
@@ -503,3 +517,302 @@ async fn all_around_tests(client: Client) -> Result<(), Box<dyn std::error::Erro
+            &Default::default(),
+        )
+        .await?;
+
+    let stats = client.get_projection_details(name.as_str(), None).await?;
+
+    assert!(stats.is_some());
+
+    let stats = stats.unwrap();
+
+    assert_eq!(stats.name, name);
+
+    Ok(())
+}
+
+// TODO - A projection must be stopped to be able to delete it. But Stop projection gRPC call doesn't not exit yet.
+async fn delete_projection(
+    client: &ProjectionClient,
+    gen_name: &mut names::Generator<'_>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let name = gen_name.next().unwrap();
+    client
+        .create_projection(
+            name.as_str(),
+            PROJECTION_FILE.to_string(),
+            &Default::default(),
+        )
+        .await?;
+
+    debug!("delete_projection: create_projection succeeded: {}", name);
+
+    let stats = client.get_projection_details(name.as_str(), None).await?;
+
+    assert!(stats.is_some());
+
+    let stats = stats.unwrap();
+
+    client.abort(name.as_str(), None).await?;
+
+    assert_eq!(stats.name, name);
+    debug!("delete_projection: reading newly-created projection statistic succeeded");
+
+    client
+        .delete(name.as_str(), false, false, false, None)
+        .await?;
+
+    debug!("delete_projection: delete projection succeeded");
+
+    Ok(())
+}
+async fn update_projection(
+    client: &ProjectionClient,
+    gen_name: &mut names::Generator<'_>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let name = gen_name.next().unwrap();
+
+    client
+        .create_projection(
+            name.as_str(),
+            PROJECTION_FILE.to_string(),
+            &Default::default(),
+        )
+        .await?;
+
+    client
+        .update(
+            name.as_str(),
+            PROJECTION_UPDATED_FILE.to_string(),
+            None,
+            None,
+        )
+        .await?;
+
+    let stats = client.get_projection_details(name.as_str(), None).await?;
+
+    assert!(stats.is_some());
+
+    let stats = stats.unwrap();
+
+    assert_eq!(stats.name, name);
+    assert_eq!(stats.version, 1);
+
+    Ok(())
+}
+
+async fn enable_projection(
+    client: &ProjectionClient,
+    gen_name: &mut names::Generator<'_>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let name = gen_name.next().unwrap();
+    client
+        .create_projection(
+            name.as_str(),
+            PROJECTION_FILE.to_string(),
+            &Default::default(),
+        )
+        .await?;
+
+    client.enable(name.as_str(), None).await?;
+
+    Ok(())
+}
+
+async fn disable_projection(
+    client: &ProjectionClient,
+    gen_name: &mut names::Generator<'_>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let name = gen_name.next().unwrap();
+    client
+        .create_projection(
+            name.as_str(),
+            PROJECTION_FILE.to_string(),
+            &Default::default(),
+        )
+        .await?;
+
+    client.enable(name.as_str(), None).await?;
+    client.disable(name.as_str(), None).await?;
+
```

Comment:
**CONCERN:** I don't see any asserts here. I think that we should verify if the changes were applied. The same concern applies to other tests without asserts.

Reply:
Done